markdown/extensions/admonition.py

```diff
@@ -80,11 +80,12 @@ def run(self, parent, blocks):
 
     def get_class_and_title(self, match):
         klass, title = match.group(1).lower(), match.group(2)
+        klass = re.sub(' +', ' ', klass)
```

Comment:
In general, I prefer (when possible) to pre-compile regex.  This would be one that can easily be either compiled once per file or at least once per class instantiation (either is probably fine). This isn't really a big issue, but every little bit helps.  Other than that, I have no complaints with anything else.

Reply:
Yes all reflex should be compiled as soon as possible.